docs/content/preview/yugabyte-voyager/migrate/bulk-data-load.md

```diff
@@ -18,6 +18,49 @@ The following page describes the steps to import data in CSV or TEXT format from
 
 * Before you perform a bulk load, in your target YugabyteDB database, create the schema of the tables into which the data in the flat files will be imported.
 
+### Prepare the target database
+
+Prepare your target YugabyteDB database cluster by creating a database, and a user for your cluster.
```

Comment:
Prepare your target YugabyteDB database cluster by creating a user for your cluster.